packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -599,6 +603,7 @@ export class BrowserCriClient {
       debug('target closed', this.currentlyAttachedTarget.targetId)
 
       await this.currentlyAttachedTarget.close().catch(() => {})
+      await this.currentlyAttachedProtocolTarget?.close().catch(() => {})
```

Comment:
Do we need to close the protocol target in the other locations where `currentlyAttachedTarget` is closed?

Reply:
Good catch. Will update